server/src/controller/user.controller.js

```diff
@@ -0,0 +1,71 @@
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const User = require("../model/user.model");
+
+exports.signup = (req, res) => {
+  const { email, password } = req.body;
+  User.find({ email }).then(user => {
+    if (user.length >= 1) {
+      return res.status(409).json({
+        message: "Email already exists"
+      });
+    } else {
+      bcrypt.hash(password, 10, (err, hash) => {
+        const user = new User({
+          email,
+          password: hash
+        });
+        user
+          .save()
+          .then(result => {
+            res.status(201).json({
+              message: "User created"
+            });
+          })
+          .catch(err => {
+            res.status(500).json({
+              error: err
+            });
+          });
+      });
+    }
+  });
+};
+
+exports.login = (req, res) => {
+  const { email, password } = req.body;
+  User.find({ email })
```

Comment:
You can use `User.findOne({ email })` here to avoid icky array notation

Reply:
Done.
(Oops, should've done that already)